packages/components/ovh-payment-method/rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig(
+  {
+    input: './src/index.js',
+  },
+  {
+    lessTildeImporter: {
+      paths: [
+        path.resolve(__dirname, 'node_modules'),
+        path.resolve(__dirname, '../../node_modules'),
+      ],
+    },
+  },
+);
+
+const outputs = [config.es()];
+
+// if (process.env.BUILD === 'production') {
```

Comment:
is this commented code required?

Reply:
yes. There is a missing commit that removed this comment block, I forget to push it :-)